R/make-traceability-matrix.R

```diff
@@ -285,11 +300,11 @@ map_tests_to_functions <- function(exports_df, pkg_source_path, verbose){
 
     if (length(uniq_funcs) == 0) {
       # probably only possible if the file is basically empty
-      return(data.frame(func = character(), test_file = character(), test_dir = character()))
+      return(tibble::tibble(func = character(), test_file = character(), test_dir = character()))
```

Comment:
I'm assuming these `data.frame -> tibble` switches here and elsewhere are just for consistency rather than being a relevant part of the fix from 8a2af23.

Reply:
Correct^